packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewerContainer.js

```diff
@@ -0,0 +1,101 @@
+import asyncStates from '@zooniverse/async-states'
+import React from 'react'
+import PropTypes from 'prop-types'
+import request from 'superagent'
+
+import LightCurveViewer from './LightCurveViewer'
+import locationValidator from '../../helpers/locationValidator'
+
+class LightCurveViewerContainer extends React.Component {
+  constructor () {
+    super()
+    
+    this.state = {
+      loading: asyncStates.initialized,
+      jsonData: null,
+    }
+    
+    //TODO: turn into variables?
+    this.width = 500
+    this.height = 500
+  }
+  
+  componentDidMount () {
+    if (this.props.subject) {
+      this.handleSubject()
+    }
+  }
+
+  componentDidUpdate (prevProps) {
+    const prevSubject = prevProps.subject
+    const { subject } = this.props
+
+    if (subject && (!prevSubject || prevSubject.id !== subject.id)) {
+      this.handleSubject()
+    }
+  }
+  
+  componentWillUnmount () {}
+
+  async handleSubject () {
```

Comment:
This method doesn't need `async`, as there's nothing waiting on it (and no `await` in the body)